nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,386 @@
+                  black_bg=False, bg_map=None, symmetric_cmap=True):
+    """Prepare info for plotting one surface map on a single mesh."""
+    info = {}
+    colors, cmin, cmax, cmap, norm, abs_threshold = colorscale(
+        cmap, surf_map, threshold, symmetric_cmap=symmetric_cmap)
+    info['inflated_left'] = to_plotly(surf_mesh)
+    info['vertexcolor_left'] = _get_vertexcolor(
+        surf_map, cmap, norm, abs_threshold, bg_map)
+    info["cmin"], info["cmax"] = float(cmin), float(cmax)
+    info['black_bg'] = black_bg
+    info['full_brain_mesh'] = False
+    info['colorscale'] = colors
+    return info
+
+
+def _check_mesh(mesh):
+    if not isinstance(mesh, str):
+        assert isinstance(mesh, collections.Mapping)
+        assert {'pial_left', 'pial_right', 'sulc_left', 'sulc_right',
+                'infl_left', 'infl_right'}.issubset(mesh.keys())
+        return mesh
+    mesh = datasets.fetch_surf_fsaverage(mesh)
+    return mesh
+
+
+def full_brain_info(volume_img, mesh='fsaverage5', threshold=None,
+                    cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+                    vol_to_surf_kwargs={}):
+    """Project 3d map on cortex; prepare info to plot both hemispheres."""
+    info = {}
+    mesh = _check_mesh(mesh)
+    surface_maps = {
+        h: surface.vol_to_surf(volume_img, mesh['pial_{}'.format(h)],
+                               **vol_to_surf_kwargs)
+        for h in ['left', 'right']
+    }
+    colors, cmin, cmax, cmap, norm, abs_threshold = colorscale(
+        cmap, np.asarray(list(surface_maps.values())).ravel(), threshold,
+        symmetric_cmap=symmetric_cmap)
+
+    for hemi, surf_map in surface_maps.items():
+        bg_map = surface.load_surf_data(mesh['sulc_{}'.format(hemi)])
+        info['pial_{}'.format(hemi)] = to_plotly(mesh['pial_{}'.format(hemi)])
+        info['inflated_{}'.format(hemi)] = to_plotly(
+            mesh['infl_{}'.format(hemi)])
+
+        info['vertexcolor_{}'.format(hemi)] = _get_vertexcolor(
+            surf_map, cmap, norm, abs_threshold, bg_map)
+    info["cmin"], info["cmax"] = float(cmin), float(cmax)
+    info['black_bg'] = black_bg
+    info['full_brain_mesh'] = True
+    info['colorscale'] = colors
+    return info
+
+
+def _fill_html_template(info, embed_js=True):
+    as_json = json.dumps(info)
+    as_html = _get_html_template().replace(
+        'INSERT_STAT_MAP_JSON_HERE', as_json)
+    as_html = add_js_lib(as_html, embed_js=embed_js)
+    return SurfaceView(as_html)
+
+
+def view_img_on_surf(volume_img, mesh='fsaverage5',
+                     threshold=None, cmap=cm.cold_hot,
+                     black_bg=False, embed_js=True):
+    """
+    Insert a surface plot of a statistical map into an HTML page.
+
+    Parameters
+    ----------
+    volume_img : Niimg-like object, 3d
+        See http://nilearn.github.io/manipulating_images/input_output.html
+
+    mesh : str or dict, optional.
```

Comment:
I haven't checked but the parameters reflected everywhere should have same name.